com.vrcfury.vrcfury/Runtime/VF/Model/StateAction.cs

```diff
@@ -34,6 +34,14 @@ public class ShaderInventoryAction : Action {
         public Renderer renderer;
         public int slot = 1;
     }
+
+    [Serializable]
+    public class PoiyomiUVTileAction : Action {
+        public Renderer renderer;
+        public int slot1 = 0;

```

Comment:
I'm not super familiar with poi's UV discard, but I think these should be x and y, or row and column, instead of slot 1 and slot 2? How are they presented in the poiyomi mat editor?

Reply:
Poiyomi shows uv tiles as rows as v= and columns as checkbooks. While in uv dissolves as alpha (row #) columns. Due to the nature of how uv meaning u=x axis and v=y axis. Could change up the internal naming as u and v to keep it consistent with practice.

Reply:
The best explanation to how UV Tile Discard works is on the Documenation Page. There's a few videos attached that can help further understand how it's set up through Blender. Take a look [here](https://www.poiyomi.com/special-fx/uv-tile-discard).